### ISSUE TITLE
Duplicated exercises in Day 4

### ISSUE THREAD

**Issue by chienchuanw** [Asabeneh/30-Days-Of-Python#565]
There are some duplicated exercises in Day 4 ([30-Days-Of-Python](https://github.com/Asabeneh/30-Days-Of-Python/tree/master)/[04_Day_Strings](https://github.com/Asabeneh/30-Days-Of-Python/tree/master/04_Day_Strings)
/04_strings.md).
1. I believe exercise 23 and exercise 26 are nearly the same.

> 23. Use index or find to **find the position of the first occurrence of the word 'because' in the following sentence: 'You cannot end a sentence with because because because is a conjunction'**

> 26. **Find the position of the first occurrence of the word 'because' in the following sentence: 'You cannot end a sentence with because because because is a conjunction'**

2. Exercise 25 and 27 are the same.

**Comment by bzm10** [comment]
Yes, Exercises 25 and 27 are exactly the same. However, Exercises 23 and 26 ask the same question but require you to use different methods.

**Comment by udayakeerthi2005** [comment]
Remove duplicate exercise in Day 4 (Exercise 27)
Hi, I noticed that Exercise 25 and Exercise 27 are identical. I would like to work on removing the duplicate to avoid confusion. Please let me know if this approach is okay.

This PR removes a duplicated exercise in Day 4.

Exercise 25 and Exercise 27 contain the exact same instruction.
This duplication may confuse learners.
Changes Made
Removed Exercise 27 as it is identical to Exercise 25.
Reason
Keeping only one version improves clarity and avoids redundancy in the exercises.
